oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -481,6 +482,14 @@ static ExternalAccountCredentials fromJson(
         .build();
   }
 
+  private static <T> T getOptional(Map<String, Object> json, String fieldName, Class<T> clazz) {
+    Object value = json.get(fieldName);
+    if (value == null || Data.isNull(value)) {
+      return null;
+    }
```

Comment:
qq, why the need for an additional `Data.isNull(value)` check?

Reply:
Oh nvm. I see after pulling this down locally.

Reply:
Not entirely sure why the value comes back as an `Object` instead of null and I'm not too familiar with the parser inside the http-client. I don't think this check adds any harm since null values from the file wouldn't have worked and this allows the client to gracefully continue for optional fields.